lib/mvi/responses/find_candidate.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+require_relative 'base'
+
+module MVI
+  module Responses
+    class FindCandidate < Base
```

Comment:
Any chance we can some informal doc here in the form of a comment?  Looks like it's responsible for parsing the MVI response but it's unclear to me what form the MVI Response is in.